sdk/communication/azure-communication-common/README.md

```diff
@@ -36,13 +36,13 @@ Add an `implementation` configuration to the `dependencies` block of your app's
 // build.gradle
 dependencies {
     ...
-    implementation "com.azure.android:azure-communication-common:1.0.2"
+    implementation "com.azure.android:azure-communication-common:1.1.0"
 }
 
 // build.gradle.kts
 dependencies {
     ...
-    implementation("com.azure.android:azure-communication-common:1.0.2")
+    implementation("com.azure.android:azure-communication-common:1.1.0")
```

Comment:
Same comment about not being able to use version `1.1.0`.

```suggestion
    implementation("com.azure.android:azure-communication-common:1.0.2")
```

Reply:
^^